### ISSUE TITLE
hide sub-folder

### ISSUE THREAD

**Issue by revoltorb** [gilbitron/Raneto#55]
Is it possible to hide a sub-folder (but still have links that go to it) from the main page? In the example below, I would want Categories 1, 2, and 3 to show up on the main page and be able to link to Sub-Category 2.a but hide Sub-Category 2.a from the TOC?

```
main content
    folder category 1
    folder category 2
        folder subcategory 2.a
    folder category 3
```

**Comment by ryanlelek** [comment]
So far the response to my comment on #18 has been well received and seems to be a solution for your use-case. The category (directory) would have a main YAML file where we could provide an option to hide the directory from the main homepage list.

Would that work well for you?

> Should we do a central YAML file per category and put all category, article, and sort metadata in there?

**Comment by revoltorb** [comment]
Yeah, if it were implemented in a YAML file that would make things much easier. Seems like it would fit my use-case perfectly.
